### PR TITLE
Core experience redirect from mutation

### PR DESCRIPTION
**Pull request by laurieboyes**
Mutating core-experience routes redirect to a safe GET-requested confirmation page instead of rendering HTML themselves

### CODE REVIEW

File: server/controllers/core-action-confirm.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+export default function (req, res) {
+
+	const actionKey = req.params.actionKey;
+	const userId = res.locals.userId;
+	const opts = req.query;
+
+	res.render('confirmation', {
+		layout: 'wrapper',
+		message: getConfirmationMessage(actionKey, userId, opts)
```
Comment (phamann) [inline server/controllers/core-action-confirm.js]: Can we throw a `404` if the action doesn't map to a predefined message?

File: client/main.scss [hunk 2]
```diff
@@ -110,15 +110,15 @@ $output-critical: true !default;
...
-			@include oTypographySansBold(xl);
-		}
 	}
 	a {
 		@include nLinksBody();
 	}
 	b {
 		@include oTypographySansBold(l);
+		@include oGridRespondTo(M) {
+			@include oTypographySansBold(xl);
```
Comment (keirog) [inline client/main.scss]: Yessss 💯 

You could take it a step further and use `oTypographySansBoldSize` here

**Comment by phamann** [comment]
:+1: LGTM 

Would be good to put more thought to a proper Redis backed flash messaging system.

**Comment by keirog** [comment]
👍

File: server/controllers/core-action-confirm.js [hunk 3]
```diff
@@ -1,13 +1,21 @@
...
 	const opts = req.query;
 
-	res.render('confirmation', {
-		layout: 'wrapper',
-		message: getConfirmationMessage(actionKey, userId, opts)
-	});
+	try {
+		res.render('confirmation', {
+			layout: 'wrapper',
+			message: getConfirmationMessage(actionKey, userId, opts)
```
Comment (keirog) [inline server/controllers/core-action-confirm.js]: Seems like calling and checking `getConfirmationMessage` first would be better, before calling render, no?

**Pull request merged**